<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jitender4568" name="com.agoda.framework.testcase.PasswordChange" tests="30" failures="4" timestamp="27 May 2019 15:25:47 GMT" time="1.751" errors="0">
  <testcase name="callerFunction" time="0.089" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.048" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.045" classname="com.agoda.framework.testcase.PasswordChange">
    <failure type="java.lang.AssertionError" message="expected [% of password is a number !!] but found [50 % of password is a number !!]">
      <![CDATA[java.lang.AssertionError: expected [% of password is a number !!] but found [50 % of password is a number !!]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.agoda.framework.util.SupportingFunctions.assertErrorAndUpdateExcel(SupportingFunctions.java:75)
at com.agoda.framework.util.SupportingFunctions.ChangePassword(SupportingFunctions.java:47)
at com.agoda.framework.testcase.PasswordChange.callerFunction(PasswordChange.java:32)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- callerFunction -->
  <testcase name="callerFunction" time="0.031" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.112" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.050" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.173" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.058" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.061" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.095" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.043" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.045" classname="com.agoda.framework.testcase.PasswordChange">
    <failure type="java.lang.AssertionError" message="expected [Password must have atleast one digit cha !!] but found [Password must have atleast one digit character !!]">
      <![CDATA[java.lang.AssertionError: expected [Password must have atleast one digit cha !!] but found [Password must have atleast one digit character !!]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.agoda.framework.util.SupportingFunctions.assertErrorAndUpdateExcel(SupportingFunctions.java:75)
at com.agoda.framework.util.SupportingFunctions.ChangePassword(SupportingFunctions.java:47)
at com.agoda.framework.testcase.PasswordChange.callerFunction(PasswordChange.java:32)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- callerFunction -->
  <testcase name="callerFunction" time="0.030" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.046" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.052" classname="com.agoda.framework.testcase.PasswordChange">
    <failure type="java.lang.AssertionError" message="expected [New password is similar to old password by more than 80%, Try ] but found [New password is similar to old password by more than 80%, Try new password]">
      <![CDATA[java.lang.AssertionError: expected [New password is similar to old password by more than 80%, Try ] but found [New password is similar to old password by more than 80%, Try new password]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.agoda.framework.util.SupportingFunctions.assertErrorAndUpdateExcel(SupportingFunctions.java:75)
at com.agoda.framework.util.SupportingFunctions.ChangePassword(SupportingFunctions.java:44)
at com.agoda.framework.testcase.PasswordChange.callerFunction(PasswordChange.java:32)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- callerFunction -->
  <testcase name="callerFunction" time="0.053" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.046" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.041" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.058" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.055" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.049" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.041" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.069" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.065" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.047" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.057" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.032" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.032" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.078" classname="com.agoda.framework.testcase.PasswordChange"/>
  <testcase name="callerFunction" time="0.050" classname="com.agoda.framework.testcase.PasswordChange">
    <failure type="java.lang.AssertionError" message="expected [P must have atleast one special character !!, Password must have atleast one uppercase character !!, Password must have atleast one digit character !!, Password have repeated character more than 4 times i.e j] but found [Password must have atleast one special character !!, Password must have atleast one uppercase character !!, Password must have atleast one digit character !!, Password have repeated character more than 4 times i.e j]">
      <![CDATA[java.lang.AssertionError: expected [P must have atleast one special character !!, Password must have atleast one uppercase character !!, Password must have atleast one digit character !!, Password have repeated character more than 4 times i.e j] but found [Password must have atleast one special character !!, Password must have atleast one uppercase character !!, Password must have atleast one digit character !!, Password have repeated character more than 4 times i.e j]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.agoda.framework.util.SupportingFunctions.assertErrorAndUpdateExcel(SupportingFunctions.java:75)
at com.agoda.framework.util.SupportingFunctions.ChangePassword(SupportingFunctions.java:47)
at com.agoda.framework.testcase.PasswordChange.callerFunction(PasswordChange.java:32)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- callerFunction -->
</testsuite> <!-- com.agoda.framework.testcase.PasswordChange -->
